static/js/components/inputs/util_test.js

```diff
@@ -0,0 +1,29 @@
+// @flow
+import { assert } from 'chai';
+
+import { dialogActions } from './util';
+
+describe('input functions', () => {
+
+  describe('dialogActions', () => {
+    it('renders properly', () => {
+      let onCancel = () => null;
+      let onSave = () => null;
+      let inFlight = true;
+      let newClass = "new-class";
+      let newText = "New Text";
+      let actions = dialogActions(onCancel, onSave, inFlight, newText, newClass);
+      assert.lengthOf(actions, 2);
+      let [cancelButton, saveButton] = actions;
+      assert.equal(cancelButton.props.className, "secondary-button cancel-button");
+      assert.equal(cancelButton.props.onClick, onCancel);
+      assert.equal(cancelButton.props.children, "Cancel");
+
+      assert.equal(saveButton.props.className, `primary-button save-button ${newClass}`);
+      assert.equal(saveButton.props.onClick, onSave);
+      assert.equal(saveButton.props.children, newText);
+      assert.equal(saveButton.props.spinning, inFlight);
+      assert.equal(saveButton.type.name, "SpinnerButton");
```

Comment:
Asserting the class name is important because it connects this unit test with the one for the `SpinnerButton`. If we swap out `SpinnerButton` with another similar component it may not have exactly the same behavior, or it may not be tested properly. This is kind of a trivial case admittedly but it seems like asserting the `spinning` prop without asserting the component we're passing the prop to would be like asserting arguments being passed to an `add` function without asserting that it is the `add` function we wanted to call